linux_os/guide/services/fapolicyd/fapolicy_default_deny/rule.yml

```
@@ -39,10 +39,14 @@ ocil: |-
 
     permissive = 0
 
-    Check that fapolicyd employs a deny-all policy on system mounts with the following command:
+    Check that fapolicyd employs a deny-all policy on system mounts with the following commands:
 
+    For RHEL 8.4 systems and older:
     $ sudo tail /etc/fapolicyd/fapolicyd.rules
 
+    For RHEL 8.5 systems and newer:
+    $ sudo tail /etc/fapolicyd/compiled.rules
```

This probably should be:
```suggestion
    $ sudo tail /etc/fapolicyd/rules.d/compiled.rules
```

Because in the fixtext the guidance is to build a file in `/etc/fapolicyd/rules.d`

---

The `/etc/fapolicyd/compiled.rules` file is correct. The rules declared in rules.d files are concatenated in this file.